ReSharper.FSharp/src/FSharp/FSharp.Psi/src/FSharp.psi

```diff
@@ -1457,6 +1457,7 @@ autoPropertyDeclaration options { stubBase="FSharpProperTypeMemberDeclarationBas
   memberKeyword{MEMBER, MemberKeyword}
   VAL<VAL, ValKeyword>
   IDENTIFIER<IDENTIFIER, Identifier>
+  typeUsage<TYPE, TypeUsage>?
```

Comment:
Isn't the type usage separated from the name with a `:`?

Reply:
Sorry, this change should not be presented in **this PR**. I will remove it.